hr_timesheet_task_required/models/account_analytic_line.py

```diff
@@ -14,6 +15,9 @@ class AccountAnalyticLine(models.Model):
         'task_id',
     )
     def _check_timesheet_task(self):
+        if config['test_enable'] and not self.env.context.get(
+                'test_hr_timesheet_task_required'):
```

Comment:
Shouldn't it be inverse? like `skip_hr_timesheet_task_required` that would by default be None/False?

Reply:
@mreficent ^

Reply:
I just copied the fix you did for `hr_timesheet_employee_required` module.

Reply:
My case is different - `hr_timesheet_employee_required` affects all modules by being present in the system (has no setting), thus I'm explicitly telling that it's ok to run tests. `hr_timesheet_task_required` on contrary does not break anything by being present, as by default it's not requiring task

Reply:
`hr_timesheet_task_required` by default is requiring task. The requirement is just the existence of the constrains.

Reply:
Ah, so sorry - that's for v11, for v12 it's https://github.com/OCA/hr-timesheet/blob/12.0/hr_timesheet_task_required/models/res_company.py#L10